components/scream/src/share/atm_process/atmosphere_process_group.cpp

```diff
@@ -86,30 +86,13 @@ AtmosphereProcessGroup (const ekat::Comm& comm, const ekat::ParameterList& param
 void AtmosphereProcessGroup::set_grids (const std::shared_ptr<const GridsManager> grids_manager) {
 
   // The atm process group (APG) acts a bit different than other atm processes.
-  // There are two catches when it comes to exposing a list of required/computed
-  // fields.
-  //  - catch 1: if (and only if) the splitting is sequential, and contains
-  //    2+ processes, it is possible that some inputs of the non-first atm
-  //    processes are computed by one (or more) of the previous ones.
-  //    In this case, such fields are not really 'requirements' of this
-  //    group, but simply "outputs" of the group (which happen to also
-  //    be reused by other atm processes in the group).
-  //  - catch 2: the APG is responsible of mapping input/output fields
-  //    from/to the reference grid. For this reason, if one of the atm
-  //    processes claims to need a field F1 which is defined on a grid other
-  //    than the reference one, the APG will expose as a requirement F2,
-  //    the same field defined on the reference grid, and will expose F1
-  //    as a 'computed' field. Similarly, if a process computes F1 on a
-  //    a grid other than the reference one, the APG will also expose
-  //    a computed field F2, the same as F1 but defined on the reference grid.
-  //    The extra computed fields will be "computed" by calling the
-  //    proper AbstractRemapper implementation for the reference-actual
-  //    grids pair. The APG stores internally a bunch of remappers,
-  //    which calls during the `run` call, to perform the remappings.
-
-  // Resizing the remappers vectors
-  m_inputs_remappers.resize(m_group_size);
-  m_outputs_remappers.resize(m_group_size);
+  // There is a catch when it comes to exposing a list of required/computed
+  // fields: if (and only if) the splitting is sequential, and contains
+  // 2+ processes, it is possible that some inputs of the non-first atm
```

Comment:
Maybe rephrase: "..., it is possible that some inputs of atmosphere processes in the list are computed by processes that precede them."

Reply:
Neat. Thanks!